lib/Constants.vala

```diff
@@ -285,6 +286,15 @@ namespace Granite {
          * Style class for non-terminal text that uses a monospace font.
          */
         public const string MONOSPACE = "monospace";
+
+        /**
+         * Style class for {@link Gtk.Label} or {@link Gtk.TextView} to emulate the appearance of Terminal. This includes
+         * text color, background color, selection highlighting, and selecting the system monospace font.
+         *
+         * When used with {@link Gtk.Label} this style includes internal padding. When used with {@link Gtk.TextView}
+         * interal padding will need to be set with {@link Gtk.Container.border_width}
```

Comment:
This documentation seems outdated now. This should probably just reference TerminalView?